src/components/Form.js

```diff
@@ -1,63 +1,80 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
```

Comment:
im not sure, but i think you dont need to import react itself as it is already part of it 😊

Reply:
Hey Jelena, thank you for checking and for your time :)